setup.py

```diff
@@ -148,27 +124,22 @@ def get_outputs(self):
 }
 
 
-def wheel_support():
+try:
+    from wheel.bdist_wheel import bdist_wheel as orig_bdist_wheel
+except ImportError:
+    pass
+else:
     class bdist_wheel(orig_bdist_wheel):
         def finalize_options(self):
             orig_bdist_wheel.finalize_options(self)
             # Mark us as not a pure python package
             self.root_is_pure = False
 
         def get_tag(self):
-            python, abi, plat = orig_bdist_wheel.get_tag(self)
+            _, _, plat = orig_bdist_wheel.get_tag(self)
             # We don't contain any python source, nor any python extensions
-            python, abi = 'py2.py3', 'none'
-            return python, abi, plat
+            return 'py2.py3', 'none', plat
 
     command_overrides['bdist_wheel'] = bdist_wheel
 
-
-try:
-    from wheel.bdist_wheel import bdist_wheel as orig_bdist_wheel
-except ImportError:
-    pass
-else:
-    wheel_support()
-
-setup(cmdclass=command_overrides)
+setup(version=f'{SHELLCHECK_VERSION}.{PY_VERSION}', cmdclass=command_overrides)
```

Comment:
here's where the version is controlled -- when tagging the repo I'd use this version :+1: